rs/nns/integration_tests/src/wait_for_quiet.rs

```diff
@@ -137,15 +144,24 @@ fn test_deadline_is_extended_with_wait_for_quiet() {
             .await
             .expect("Error calling the manage_neuron api.");
 
-        let pi: Option<ProposalInfo> = nns_canisters
+        let final_proposal_info: Option<ProposalInfo> = nns_canisters
             .governance
             .query_("get_proposal_info", candid, (pid.id,))
             .await
             .unwrap();
 
-        let final_deadline = pi.unwrap().deadline_timestamp_seconds.unwrap();
+        let final_deadline = final_proposal_info
+            .as_ref()
+            .unwrap()
+            .deadline_timestamp_seconds
+            .unwrap();
 
-        assert!(final_deadline > initial_deadline);
+        assert!(
+            final_deadline > initial_deadline,
+            "{:?}\n{:#?}",
+            initial_deadline,
+            final_proposal_info,
```

Comment:
dump ALL the things!

Because every crime detective would LOVE to have a snapshot of the criminal act. Making crime solving dang near trivial is what we should be striving for (and it isn't as hard as real life crime solving, because we can much more easily put cameras EVERYWHERE).